share/spice/forvo/forvo.js

```diff
@@ -159,3 +50,26 @@ Handlebars.registerHelper("forvo_sex", function(sex) {
     return "Female";
 });
 
+Handlebars.registerHelper('ifCond', function (v1, operator, v2, options) {
```

Comment:
@alexbaforvo we actually have an `{{#and}}` helper that checks if the input variables are defined, and if so, executes its block:

```
{{#and foo bar}}
foo and bar are defined!
{{/and}}
```